glide-core/redis-rs/redis/src/cmd.rs

```diff
@@ -204,14 +204,14 @@ fn args_len<'a, I>(args: I, cursor: u64) -> usize
 where
     I: IntoIterator<Item = Arg<&'a [u8]>> + ExactSizeIterator,
 {
-    let mut totlen = 1 + countdigits(args.len()) + 2;
+    let mut total_len = 1 + countdigits(args.len()) + 2;
```

Comment:
we should avoid direct math operations and use `checked_add` or `saturating_add`